src/index.html

```diff
@@ -7,12 +7,69 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__wrapper-img">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Monitor IMac"
```

Comment:
``alt`` attribute should describe the image content.

![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/46fb0bec-dc95-478c-a7be-54599acb4e71)